src/app/plugins/modreport.plugin.ts

```diff
@@ -10,11 +10,25 @@ export class ModReportPlugin extends Plugin {
   public permission: ChannelType = ChannelType.Staff;
   public pluginChannelName: string = Constants.Channels.Staff.UserOffenses;
 
+  private _commadRegex: RegExp = /(add|list)\s+([^#]+#\d{4})\s*(.*)/;
+
   constructor(public container: IContainer) {
     super();
   }
 
   public async execute(message: IMessage, args: string[]) {
+    const full_arg = args.join(' ');
+    const match_arr = full_arg.match(this._commadRegex);
+
+    if (!match_arr) {
+      message.reply('Invalid syntax.');
+      return;
+    }
+
+    args[0] = match_arr[1];
```

Comment:
do we really need to mutate `args` here? i'm leaning towards the side of creating separate variables with more descriptive names

Reply:
Agree. Will fix